src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -725,19 +768,22 @@ bulletListItemToMarkdown opts bs = do
 -- | Convert ordered list item (a list of blocks) to markdown.
 orderedListItemToMarkdown :: PandocMonad m
                           => WriterOptions -- ^ options
-                          -> Text        -- ^ list item marker
+                          -> Text          -- ^ list item marker
                           -> [Block]       -- ^ list item (list of blocks)
                           -> MD m (Doc Text)
 orderedListItemToMarkdown opts marker bs = do
   let exts = writerExtensions opts
   contents <- blockListToMarkdown opts $ taskListItemToAscii exts bs
+  variant <- asks envVariant
   let sps = case writerTabStop opts - T.length marker of
                    n | n > 0 -> literal $ T.replicate n " "
                    _ -> literal " "
   let ind = if isEnabled Ext_four_space_rule opts
                then writerTabStop opts
                else max (writerTabStop opts) (T.length marker + 1)
-  let start = literal marker <> sps
+  let start = case variant of
+              Markua -> literal marker
```

Comment:
Don't you want at least one space after the list marker?

Reply:
Yes, the space was already passed along with the marker. We changed this slightly to improve the readability/understanding.